packages/compass-aggregations/src/modules/index.ts

```diff
@@ -1,176 +1,43 @@
+import autoPreview from './auto-preview';
+import id from './id';
+import savedPipeline from './saved-pipeline';
+import importPipeline from './import-pipeline';
+import appRegistry from '@mongodb-js/mongodb-redux-common/app-registry';
+import settings from './settings';
+import savingPipeline from './saving-pipeline';
+import outResultsFn from './out-results-fn';
+import projections from './projections';
+import isNewPipelineConfirm from './is-new-pipeline-confirm';
+import updateViewError from './update-view';
+import aggregation from './aggregation';
+import countDocuments from './count-documents';
+import explain from './explain';
+import isDataLake from './is-datalake';
+import workspace from './workspace';
 import aggregationWorkspaceId from './aggregation-workspace-id';
-import indexes, {
-  INITIAL_STATE as INDEXES_INITIAL_STATE,
-} from './indexes';
-
-import { createLoggerAndTelemetry } from '@mongodb-js/compass-logging';
-
-const { debug } = createLoggerAndTelemetry('COMPASS-AGGREGATIONS-UI');
-
-import { getDirectory } from '../utils/get-directory';
-import { PipelineStorage } from '../utils/pipeline-storage';
-
-import type { Projection } from './pipeline';
-
-/**
- * The intial state of the root reducer.
- */
-export const INITIAL_STATE = {
-  appRegistry: APP_REGISTRY_STATE,
-  dataService: DS_INITIAL_STATE,
-  fields: FIELDS_INITIAL_STATE,
-  inputDocuments: INPUT_INITIAL_STATE,
-  namespace: NS_INITIAL_STATE,
-  env: ENV_INITIAL_STATE,
-  isTimeSeries: IS_TIME_SERIES_INITIAL_STATE,
-  serverVersion: SV_INITIAL_STATE,
-  pipeline: PIPELINE_INITIAL_STATE,
-  savedPipeline: SP_INITIAL_STATE,
-  restorePipeline: RESTORE_PIPELINE_STATE,
-  name: NAME_INITIAL_STATE,
-  collationString: COLLATION_STRING_INITIAL_STATE,
-  isAtlasDeployed: IS_ATLAS_DEPLOYED_INITIAL_STATE,
-  isReadonly: IS_READONLY_INITIAL_STATE,
-  comments: COMMENTS_INITIAL_STATE,
-  sample: SAMPLE_INITIAL_STATE,
-  autoPreview: AUTO_PREVIEW_INITIAL_STATE,
-  id: ID_INITIAL_STATE,
-  isModified: IS_MODIFIED_INITIAL_STATE,
-  importPipeline: IMPORT_PIPELINE_INITIAL_STATE,
-  settings: SETTINGS_INITIAL_STATE,
-  limit: LIMIT_INITIAL_STATE,
-  largeLimit: LARGE_LIMIT_INITIAL_STATE,
-  maxTimeMS: MAX_TIME_MS_INITIAL_STATE,
-  savingPipeline: SAVING_PIPELINE_INITIAL_STATE,
-  projections: PROJECTIONS_INITIAL_STATE as Projection[],
-  outResultsFn: OUT_RESULTS_FN_INITIAL_STATE as null | ((namespace: string) => void),
-  editViewName: EDIT_VIEW_NAME_INITIAL_STATE,
-  sourceName: SOURCE_NAME_INITIAL_STATE,
-  isNewPipelineConfirm: IS_NEW_PIPELINE_CONFIRM_STATE,
-  updateViewError: UPDATE_VIEW_ERROR_INITIAL_STATE,
-  aggregation: AGGREGATION_INITIAL_STATE,
-  workspace: WORKSPACE_INITIAL_STATE,
-  countDocuments: COUNT_INITIAL_STATE,
-  explain: EXPLAIN_INITIAL_STATE,
-  isDataLake: DATALAKE_INITIAL_STATE,
-  indexes: INDEXES_INITIAL_STATE,
-  pipelineBuilder: PIPELINE_BUILDER_INITIAL_STATE,
-};
+import indexes from './indexes';
 
 /**
  * Reset action constant.
```

Comment:
Should be safe to remove this (these) constants?

Reply:
Yes, thanks!

Reply:
Accidentally brought them back in merge conflict resolution, thanks again for catching, fixed in https://github.com/mongodb-js/compass/pull/3559/commits/c22d7cd8dd3156cdc5b2360f58d2b7036e38d8b0